src/openloco/localisation/stringmgr.h

```diff
@@ -5,6 +5,11 @@
 namespace openloco
 {
     using string_id = uint16_t;
+
+    namespace string_ids
+    {
+        constexpr string_id null = 0xFFFF;
```

Comment:
Should this perhaps go with the other constants in string_ids.h?

Reply:
It was but I moved it because it would make more things than necessary import all of the IDs when it only needed null.